sycl/unittests/assert/assert.cpp

```diff
@@ -528,54 +368,3 @@ TEST(Assert, TestAssertServiceKernelHidden) {
 
   EXPECT_TRUE(NoFoundServiceKernelID);
 }
-
-TEST(Assert, TestInteropKernelNegative) {
```

Comment:
These interop tests was seg-faulting as the interop sycl apis was now using opencl entry points to query the dependency opencl handles. 

These tests was mocking an opencl handle that is not valid, once this invalid cl handle is passed to the sycl interop api, it will start query about the dependency handles, like the cl_kernel here, we need to query about the cl_program associated with it, so we will use `clGetKernelInfo` and once this cl entry is called without a valid cl handle, opencl will seg-fault and the entire program. So, The solution for this tests was to add a valid cl_kernel handle, which is also a problem as these tests CI pipeline doesn't use any hardware resource unlike the e2e tests, also adding a valid cl handle will make them more of an e2e test than a unittest. So, these tests where moved to e2e tests.

Reply:
With moving this test to the e2e test, it was already covered there in this [test](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Basic/interop/construction_ocl.cpp). As this test was only testing that the assert will not be called and the program will finish successfully, the interop tests in e2e tests will fail if any assert was raised.